src/cfitsio.jl

```diff
@@ -502,12 +553,8 @@ function fits_read_col{T}(f::FITSFile,
                           firstelem::Integer,
                           data::Array{T})
 
-    anynull = Int32[0]
-    status = Int32[0]
-
-    firstrow64 = convert(Int64, firstrow)
-    firstelem64 = convert(Int64, firstelem)
-    nelements64 = convert(Int64, length(data))
```

Comment:
Should there be a line `nelements = length(data)` here? (`nelements` is used below in ccall)